civis/io/_tables.py

```diff
@@ -55,9 +56,10 @@ def read_civis(table, database, columns=None, use_pandas=False,
         will be used.
     polling_interval : int or float, optional
         Number of seconds to wait between checks for query completion.
-    archive : bool, optional
-        If ``True`` (the default), archive the export job as soon as it
-        completes.
+    archive : bool, optional (deprecated)
+        If ``True``, archive the import job as soon as it completes.
```

Comment:
Would it be more clear here to say that archive is just ignored in favor of hidden?